frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -70,33 +68,32 @@ export const detectRGW: FeatureDetector = async (dispatch) => {
 
 export const detectOCS: FeatureDetector = async (dispatch) => {
   try {
-    const storageCluster = await fetchK8s(
-      OCSServiceModel,
-      OCS_INTERNAL_CR_NAME,
-      CEPH_STORAGE_NAMESPACE,
-    );
+    const storageClusters = await k8sList(OCSServiceModel, { ns: CEPH_STORAGE_NAMESPACE });
+    const storageCluster = storageClusters?.[0];
+    const isInternal = _.isEmpty(storageCluster.spec.externalStorage);
```

Comment:
Is the `spec.externalStorage` always present ?
If not, then `_,isEmpty` will return `true`  when `spec.externalStorage === undefined`.
If, yes, then I guess it will be always `{}` for attached devices and internal mode ?

Reply:
yes.